### DISCUSSION TITLE
【zhoujun20是谁？】更改predict_cls.py中path后，调用出现core dump

### DISCUSSION THREAD

**Discussion by CheungZeeCn** [PaddlePaddle/PaddleOCR#12502] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：docker, ubuntu 20.04.5
- 版本号/Version：Paddle：paddlepaddle-gpu 2.5.1  PaddleOCR： 2.6.1.3 
由于 tools 路径冲突，我在 predict_cls.py 的开头中，加入了一行sys.path.insert(0, os.path.abspath(os.apth.join(__dir__, '../..'))
- 问题相关组件/Related components： ch_pp-OCRv4_det/rec, ppocr_mobile_v2.0_cls
- 运行指令/Command Code:  line 111 of predict_cls.py:  self.predictor.run()
- 完整报错/Complete Error Message：

抱歉内部系统无法拷贝文字，只能截图:
<img width="961" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/2025362/7dfe6b28-9ad1-4cf5-a86b-bb337c3d253e">

<img width="1003" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/2025362/153d1b23-89e4-4d2f-ae5c-baa16375a995">

**Comment by rockyzhengwu** [comment]
这些是从 pmodel 文件里出来的，不过我也不知道为什么 pmodel 文件里为什么会有这些。
vim ~/.paddleocr/whl/cls/ch_ppocr_mobile_v2.0_cls_infer/inference.pdmodel 
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/10402091/8e063d51-b617-4b8b-be31-f572f6b1b7de)
